plugins/Installation/lang/en.json

```diff
@@ -19,145 +19,145 @@
         "HappyAnalysing": "Happy analysing!",
         "Installation": "Installation",
         "InformationalResults": "Informational results",
-        "CopyBelowInfoForSupport": "Copy or download the below info in case our support team asks you for this information.",
+        "CopyBelowInfoForSupport": "Copy or download the info below, in case our support team asks you for this info.",
         "CopySystemCheck": "Copy system check",
         "DownloadSystemCheck": "Download system check",
         "InstallationStatus": "Installation status",
-        "InsufficientPrivilegesHelp": "You can add these privileges by using a tool such as phpMyAdmin or by executing the right SQL queries. If you don't know how to do these things, please ask your sysadmin to grant these privileges for you.",
-        "InsufficientPrivilegesMain": "Either the Database does not exist (and could not be created), or the specified User has insufficient privileges. Database user must have the following privileges: %s",
+        "InsufficientPrivilegesHelp": "These privileges can be added by phpMyAdmin, or by executing the right SQL queries. If you don't know how, please ask your sysadmin to do so.",
+        "InsufficientPrivilegesMain": "Either the database does not exist (and could not be created), or the specified user has insufficient privileges. A database user must have the following privileges: %s",
         "InvalidStateError": "Error: Matomo is already installed. %1$s Go back %2$s to Matomo%3$s.",
-        "JsTagArchivingHelp1": "For medium and high traffic websites there are certain optimizations that should be made to help Matomo run faster (such as %1$ssetting up auto-archiving%2$s).",
+        "JsTagArchivingHelp1": "For medium and high traffic website certain optimizations help Matomo run faster (such as %1$ssetting up auto-archiving%2$s).",
         "JSTracking_EndNote": "Note: After the installation process, you can generate customized tracking code in the %1$sTracking Code%2$s admin section.",
-        "JSTracking_Intro": "To track your web traffic with Matomo you need to make sure some extra code is added to each of your webpages.",
+        "JSTracking_Intro": "To track your web traffic with Matomo you need to make sure a tracking code is added to each of your webpages.",
         "LargePiwikInstances": "Help for users with high traffic websites",
         "Legend": "Legend",
-        "LoadDataInfileRecommended": "If your Matomo server tracks high traffic websites (eg. > 100,000 pages per month), we recommend trying to fix this problem.",
-        "LoadDataInfileUnavailableHelp": "Using %1$s will greatly speed Matomo's archiving process up. To make it available to Matomo, try updating your PHP & MySQL software and make sure your database user has the %2$s privilege.",
+        "LoadDataInfileRecommended": "Try fixing this problem if your Matomo server tracks high traffic websites (e.g. > 100,000 pages per month).",
```

Comment:
Suggest reverting as existing is more explicit.

Reply:
Who is "we"?
"eg." is an error.
It should be "You should", but I didn't want to change it.

Reply:
```suggestion
        "LoadDataInfileRecommended": "You should fix this problem if your Matomo server tracks high-traffic websites (e.g. > 100,000 pages per month).",
```